### DISCUSSION TITLE
BIOS Raid for PCI to NVME adapter

### DISCUSSION THREAD

**Discussion by vvavepacket** [madMAx43v3r/chia-plotter#542] [Q&A]
I purchased 2 pcs [PCI to NVME adapter](https://www.amazon.com/gp/product/B07JCFHCVN/ref=ppx_od_dt_b_asin_title_s00?ie=UTF8&psc=1)   and I wanted to know if I can BIOS raid this?

I went to my BIOS and it could only detect NVME plug into the M2 socket... How can I make them detect those plug to the PCIe sockets?

My mobo is MSI Gaming Plus X570

**Comment by cyperbg** [comment]
Do you not see them at all in the BIOS?

**Comment by vvavepacket** [comment]
I'm seeing them now



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
06/17/21,
07:02:31 PM
